subgraphs/balancer-forks/src/mappings/balancerTokenAdminMappings.ts

```diff
@@ -0,0 +1,25 @@
+import { BigDecimal } from "@graphprotocol/graph-ts";
+import { MiningParametersUpdated } from "../../generated/BalancerTokenAdmin/BalancerTokenAdmin";
+import * as constants from "../common/constants";
+import { getOrCreateRewardToken } from "../common/initializers";
+import { getRewardsPerDay } from "../common/rewards";
+
+export function handleMiningParametersUpdated(
+  event: MiningParametersUpdated
+): void {
+  let protocolToken = getOrCreateRewardToken(
+    constants.PROTOCOL_TOKEN_ADDRESS,
+    constants.RewardTokenType.DEPOSIT,
+    event.block
+  );
+  protocolToken._inflationRate = BigDecimal.fromString(
+    event.params.rate.toString()
+  );
```

Comment:
shouldn't we divide this number by `pow(10, 18)`?

Reply:
It looks like you are setting the rewardTokenEmissionsAmount to the amount before decimal upscaling since it is in the full token about in the constants. The base for rewardTokenEmissionsAmount should be #tokens * 10^token.decimals. So I thought this was correct. 

<img width="619" alt="Screen Shot 2022-08-20 at 11 59 55 PM" src="https://user-images.githubusercontent.com/56660047/185776279-b2387890-3d34-4359-bb7c-0de0ea3ac6cd.png">
<img width="615" alt="Screen Shot 2022-08-20 at 11 54 43 PM" src="https://user-images.githubusercontent.com/56660047/185776182-79ad30e9-36fb-44a1-83ea-4c4389793949.png">
<img width="717" alt="Screen Shot 2022-08-20 at 11 55 15 PM" src="https://user-images.githubusercontent.com/56660047/185776183-70509eea-0104-474c-bade-6709e66a8030.png">

Reply:
I thought the FE ([subgraphs.xyz](https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/name/messari/balancer-v2-ethereum&tab=pool&poolId=0x32296969ef14eb0c6d29669c550d4a0449130230)) doesn't handle the decimals manipulation for `rewardTokenEmissionAmoiunt`. So I normally divide the rewards emission per day by the rewards decimals `pow(10, 18)`.